src/client/CourseRequestForm/index.js

```diff
@@ -0,0 +1,116 @@
+                <Button>Add additional Class ID</Button>
+            </Table.Cell>
+            <Table.Cell>{course.timeRequested}</Table.Cell>
+            <Table.Cell>
+                <TextInput width='200px' type='email' renderLabel=''/>
+            </Table.Cell>
+            <Table.Cell>{course.status}</Table.Cell>
+            <Table.Cell>
+                <Flex justifyItems='center'>
+                    <Flex.Item>
+                        <Checkbox label='' size='small'/>
+                    </Flex.Item>
+                </Flex>
+            </Table.Cell>
+            <Table.Cell>
+                <Flex justifyItems='center'>
+                    <Flex.Item>
+                        <Checkbox label='' size='small' defaultChecked/>
+                    </Flex.Item>
+                </Flex>
+            </Table.Cell>
+            <Table.Cell>
+                <Flex justifyItems='center'>
+                    <Flex.Item>
+                        <Checkbox label='' size='small' defaultChecked/>
+                    </Flex.Item>
+                </Flex>
+            </Table.Cell>
+        </Table.Row>
+    ));
+    return (
+        <FormFieldGroup label='' description=''>
+            <TextInput width='400px' type='email' renderLabel='Email to contact when these courses are built:'/>
+            <Checkbox onChange={handleEmailToggle} size='small' label='Toggle email instructors' />
+            <CheckboxGroup
+                description='Build filters:'
+                layout='columns'
+                defaultValue={['ugrad', 'grad', 'tut']}
+                name='buildfilters'
+                size='small'
+            >
+                <Checkbox label='ugrad' value='ugrad' />
+                <Checkbox label='grad' value='grad' />
+                <Checkbox label='tut' value='tut' />
+            </CheckboxGroup>
+            <Table caption=''>
+                <Table.Head>
+                    <Table.Row>
+                        <Table.ColHeader id='requestID'>Request ID</Table.ColHeader>
+                        <Table.ColHeader id='term'>Term</Table.ColHeader>
+                        <Table.ColHeader id='classID'>Class ID</Table.ColHeader>
+                        <Table.ColHeader id='department'>Department</Table.ColHeader>
+                        <Table.ColHeader id='course'>Course</Table.ColHeader>
+                        <Table.ColHeader id='crosslistedclassIDs'>Crosslisted Class IDs</Table.ColHeader>
+                        <Table.ColHeader id='timerequested'>Time requested</Table.ColHeader>
+                        <Table.ColHeader id='requestoremail'>Requestor Email</Table.ColHeader>
+                        <Table.ColHeader id='status'>Status</Table.ColHeader>
+                        <Table.ColHeader id='emailinstructors'>Email instructors</Table.ColHeader>
+                        <Table.ColHeader id='donotsendurl'>Do NOT send URL to MyUCLA</Table.ColHeader>
```

Comment:
@kubilayagi I never liked the negative wording of this column. Can we have it say and function as: "Send class link to MyUCLA"

Reply:
@rlorenzo done